tests/ftests/090-cgcreate-non-default-controllers-v2.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# cgcreate non default controller functionality test
+#
+# Copyright (c) 2023 Oracle and/or its affiliates.
+# Author: Kamalesh Babulal <kamalesh.babulal@oracle.com>
+#
+
+from cgroup import Cgroup, Mode
+import consts
+import ftests
+import sys
+import os
+
+CONTROLLERS = ['hugetlb', 'misc']
+
+CGNAME = '090cgcreate'
+CGNAME1 = '{}/childcg'.format(CGNAME)
```

Comment:
nitpick - you could use `os.path.join()` here.

Reply:
@drakenclimber Thank you so much for the review.  Sure, will use `os.path.join()` here.